extensions/sql-database-projects/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sql-database-projects",
   "displayName": "SQL Database Projects",
   "description": "The SQL Database Projects extension for Azure Data Studio allows users to develop and publish database schemas.",
-  "version": "0.10.0",
+  "version": "0.11.0",
   "publisher": "Microsoft",
   "preview": true,
   "engines": {
```

Comment:
do we need to update Azdata version too?

Reply:
yeah I think this needs to be updated since there was a change in the data workspace extension to changing projectActions to projectToolbarActions in IProjectProvider in #15504, so the dashboard might not work with older ADS versions